advanced_functionality/scikit_learn_bring_your_own_LightGBM_model/scikit_learn_bring_your_own_LightGBM_model.ipynb

```diff
@@ -0,0 +1,1121 @@
+{
```

Comment:
<p>Add a brief explanation of what these results mean / how to interpret them.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3084/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>I will do that.</p>

Reply:
Done.